data/json/items/tools.json

```diff
@@ -264,6 +264,7 @@
     "description": "A homemade collapsible telescope. Too large and inaccurate to use as a weapon scope, but carrying this item in your inventory will double the distance that is mapped around you during your travels.",
```

Comment:
I know it's not your changes but could you put double space after a period since you're here?